core/src/test/java/com/yandex/yoctodb/FieldValueExtractionTest.java

```diff
@@ -43,6 +47,10 @@ private DocumentProvider build(
                                 FIELD_NAME,
                                 0,
                                 indexOption)
+                        .withField(INT_FIELD_NAME, 25, indexOption)
+                        .withField(LONG_FIELD_NAME, 25L, indexOption)
+                        .withField(SHORT_FIELD_NAME, (short) 25, indexOption)
+                        .withField(CHAR_FIELD_NAME, 'a', indexOption)
```

Comment:
Please, move all of this literals to constants

Reply:
Fixed